packages/dcos-integration-test/extra/conftest.py

```diff
@@ -1,18 +1,37 @@
 import logging
 import os
 
-import api_session_fixture
 import pytest
-from dcos_test_utils import logger
 from test_dcos_diagnostics import (
     _get_bundle_list,
     check_json,
     wait_for_diagnostics_job,
     wait_for_diagnostics_list
 )
-logger.setup(os.getenv('TEST_LOG_LEVEL', 'INFO'))
+from test_helpers import expanded_config
+
 log = logging.getLogger(__name__)
 
+pytest_plugins = ['pytest-dcos']
```

Comment:
I think, this is a secret sauce for pytest to register the plugin `pytest-dcos`. A comment above could help the reader.

Reply:
I think this is standard. It's at the top of the "how to use plugins" page on the pytest docs https://docs.pytest.org/en/latest/plugins.html#requiring-loading-plugins-in-a-test-module-or-conftest-file